terraform/environments/prd/aws/vpc_variables.tf

```diff
@@ -125,4 +125,79 @@ locals {
       }
     }
   }
+
+  asgs = {
+    asg_sg = {
+      ingress_443 = {
+        from_port   = 443
+        to_port     = 443
+        protocol    = "tcp"
+        cidr_blocks = ["10.0.4.0/24", "10.0.5.0/24", "10.0.6.0/24"]
+        }
+      ingress_80 = {
+        from_port  = 80
+        to_port    = 80
+        protocol   = "tcp"
+        cidr_blocks = ["10.0.4.0/24", "10.0.5.0/24", "10.0.6.0/24"]
+      }
+      egress = {
+        from_port   = 0
+        to_port     = 0
+        protocol    = "-1"
+        cidr_blocks = ["10.0.4.0/24", "10.0.5.0/24", "10.0.6.0/24"]
+      }
+    }
+    asgs_services = {
+      asg_ivanoff_authn_tf = {
+        lt = {
+          name = "lt-authn-tf"
+          path = "./files/cloud-configs/authn.yaml"
+          instance_type = "t3.micro"
+          git_branch = "features-hw-24-26"
+          secret = ""
+        }
+        asg = {
+          name = "asg-authn-tf"
+          availability_zones = ["eu-north-1b", "eu-north-1c"]
+          desired_capacity = 2
+          min_size = 2
+          max_size = 4
+        }
+      }
+
+      asg_ivanoff_authz_tf = {
+        lt = {
+          name = "lt-authz-tf"
+          path = "./files/cloud-configs/authz.yaml"
+          instance_type = "t3.micro"
+          git_branch = "feature-hw-26"
+          secret = ""
+        }
+        asg = {
+          name = "asg-authz-tf"
+          availability_zones = ["eu-north-1a", "eu-north-1c"]
+          desired_capacity = 2
+          min_size = 2
+          max_size = 4
+        }
+      }
+
+      asg_ivanoff_bank_tf = {
+        lt = {
+          name = "lt-bank-tf"
+          path = "./files/cloud-configs/bank.yaml"
+          instance_type = "t3.micro"
+          git_branch = "feature-hw-26"
+          secret = ""
```

Comment:
so what is the plan here with this secret it will not work right?

Reply:
> so what is the plan here with this secret it will not work right?

I don't quite understand the question. What do u mean?
Without a secret or with an empty string it works. If there is a secret, then it must be indicated in vars.

Reply:
i fixed this